kickstarts/base.ks.erb

```diff
@@ -3,6 +3,7 @@
 logging --level=debug
 
 #version=DEVEL
+text
```

Comment:
We didn't have this before by the way. If you build without "text", and look at the console of the VM being built (virsh list; virt-viewer <vmid>), then you see that we're running the graphical installer.